Enhance data visualization attributes section with more images (#51)
fhdsl/DataViz_Considerations: 03_data.qmd

**Labels** include axis labels, titles, and direct annotations within the plot perhaps specifying a specific summary statistic/number or pointing out the name/value of a specific data point or line. They may be combined with dashed lines, colors, or other visual channels to provide further clarity.

```{r, fig.align = 'center', out.width = "100%", echo = FALSE, fig.alt = ""}
ottrpal::include_slide("https://docs.google.com/presentation/d/1fu-KfdN2ldOXB49o9zdpurFbVvl1bOh8cfQ3vu0szk4/edit?slide=id.g3b3602af4d0_0_25#slide=id.g3b3602af4d0_0_25")
```

> Legends are a form of labels. In some cases, directly labeling groups on a plot is prefered over using a legend. This is especially true if the data groups are fairly well separated on the plot or you can visually associate the labels with whatever channel is beieng used to represent the groups (e.g., color).

Author: ```suggestion
Legends are a form of labels. In some cases, directly labeling groups on a plot is preferred over using a legend. This is especially true if the data groups are fairly well separated on the plot or you can visually associate the labels with whatever channel is being used to represent the groups (e.g., color).
```

> Legends are a form of labels. In some cases, directly labeling groups on a plot is preferred over using a legend. This is especially true if the data groups are fairly well separated on the plot or you can visually associate the labels with whatever channel is being used to represent the groups (e.g., color).

Reviewer: might be worth noting that this could be a good option if you have too many colors for viewers to distinguish - although faceting could also help

Author: Good idea!

```{r, fig.align = 'center', out.width = "100%", echo = FALSE, fig.alt = ""}
ottrpal::include_slide("https://docs.google.com/presentation/d/1fu-KfdN2ldOXB49o9zdpurFbVvl1bOh8cfQ3vu0szk4/edit?slide=id.g3f2e7f70982_0_1217#slide=id.g3f2e7f70982_0_1217")
```

#### Subplots

> **Subplots**, also known as faceting, subplots separates out the data into different panels based on group membership. The same plot type will be utilized in each panel. Panels may or may not use the same scaling and axis limits. This is especially useful when the data would overlap within the same plot / panel or there are a lot of groups with a lot of data within each group. Faceting may be horizontal or vertical -- panels should be arranged depending on which comparison between groups is most appropriate. 

Reviewer: ```suggestion
**Subplots**, also known as faceting, subplots separate out the data into different panels based on group membership. The same plot type will be utilized in each panel. Panels may or may not use the same scaling and axis limits. This is especially useful when the data would overlap within the same plot / panel or there are a lot of groups with a lot of data within each group. Faceting may be horizontal or vertical -- panels should be arranged depending on which comparison between groups is most appropriate. 
```

  * The polar coordinate system uses angles to position groups and radii length to represent magnitude or group size.
  * A projected coordinate system is a two dimensional map.

```{r, fig.align = 'center', out.width = "100%", echo = FALSE, fig.alt = ""}

> ottrpal::include_slide("https://docs.google.com/presentation/d/1fu-KfdN2ldOXB49o9zdpurFbVvl1bOh8cfQ3vu0szk4/edit?slide=id.g3f2e7f70982_0_1334#slide=id.g3f2e7f70982_0_1334"

Author: ```suggestion
ottrpal::include_slide("https://docs.google.com/presentation/d/1fu-KfdN2ldOXB49o9zdpurFbVvl1bOh8cfQ3vu0szk4/edit?slide=id.g3f2e7f70982_0_1334#slide=id.g3f2e7f70982_0_1334")
```
